test/built-ins/GeneratorPrototype/return/try-finally-set-property-within-try.js

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2022 Bo Pang. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-generator.prototype.return
+description: >
+    When a generator is paused within a `try` block of a `try..finally`
+    statement, `return` should interrupt control flow as if a `return`
+    statement had appeared at that location in the function body.
+    The `finally` block is still evaluated, and may override the return value.
+features: [generators]
+---*/
+
+
+var obj={foo:undefined};
+function* g() {
+  try {
+    obj.foo=yield;
+  } finally {
+    return 1;
+  }
+}
+var iter = g();
+var result;
+
+iter.next();
+result = iter.return(45).value;
+assert.sameValue(obj.foo,undefined,'`obj.foo` did not set');
+assert.sameValue(result,1,'return in finally executes first');
```

Comment:
```suggestion
assert.sameValue(obj.foo,'not modified','`obj.foo` did not set');
assert.sameValue(result,1,'finally block supersedes return value');
```

I updated the message above to reflect the actual behavior, PTAL.

Reply:
I agree with @leobalter , and would also add spaces to match the prevailing style.
```suggestion
assert.sameValue(obj.foo, 'not modified', '`obj.foo` was not set');
assert.sameValue(result, 1, 'finally block supersedes return value');
```